pages/api/opprtunities/[id].ts

```diff
@@ -2,19 +2,20 @@ import { prisma } from "@/lib";
 import { NextApiRequest, NextApiResponse } from "next";
 
 
-export default async function handler(req:NextApiRequest, res:NextApiResponse) {
-    const {id}:Partial<{ [key: string]: string | string[]; }> = req.query
-    try{
-        if(typeof id === 'string'){
-            const job:CrawledOpportunity = await prisma.crawledOpportunity.findUniqueOrThrow({
-                where:{
-                    id: id
-                }
-            });
-            return res.status(200).json(job)
-        }
-  }
-  catch(err){
-    return res.status(500).json({message:"there's smth went wrong",Error:err})
-  }
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+    const id = String(req.query.id);
```

Comment:
this need to control job, like deleting a job, updating a job, listing a job

so the route is a singular route, not like the opportunities route, which should be considered to be plural